templates/components/telephone.build.hbs

```diff
@@ -1,10 +1,11 @@
 <input
   type="tel"
   class="form-control"
-  v-model.trim="value"
-  v-on:input="updateValue()"
+  v-model.trim.lazy="value"
+  v-on:blur="updateValue()"
   :name="name"
   :id="name"
   :placeholder="placeholder"
-  :required="required"
 />
+<p class="text-danger" v-if="$v.value.required === false && $v.value.$dirty">Field is required.</p>
+<p class="text-danger" v-if="$v.value.phone === false && $v.value.$dirty">Phone could contain <b>; , . ( ) - + SPACE * #</b> and numbers.</p>
```

Comment:
> Phone can only contain numbers and the <b>; , . ( ) - + SPACE * #</b> characters.

Note: why allowing `;` and `,`?

Reply:
Because @tonytlwu live a comment in this doc about characters are also allowed https://docs.google.com/document/d/18aJgWz_8E8I0NbWPO4EcKGwK1xc8tX7E-voXVh-e5Qg/edit 
<img width="1145" alt="Снимок экрана 2019-07-16 в 20 49 47" src="https://user-images.githubusercontent.com/29350776/61317252-4d824380-a80b-11e9-98f4-34fc35326264.png">

Reply:
Ok, thanks for confirming